app/models/work.rb

```diff
@@ -0,0 +1,41 @@
+class Work < ApplicationRecord
+  # QUESTION: how to get this to work? CATEGORIES = ["album", "book", "movie"]
+  has_many :votes, dependent: :delete_all
+
+  validates :title, presence: true, uniqueness: {scope: :category, message: "This category already knows about that title." }
+  # TODO: validates :publication_year --> how to get year only??
+
+  # TODO: necessary???? validates_inclusion_of :category, :in => CATEGORIES
```

Comment:
I would say the inclusion validation would be important. If we've built our views correctly then it shouldn't be possible for a user to send in an invalid category, but it would be a good safety check for yourself to make sure you've wired things up correctly. For example, if you misspelled "ablum" in the category dropdown menu, this would catch it.